modules/agent-host-roles.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="agent-host-roles_{context}"]
+= Host roles
+
+Each host in the cluster is assigned a role of either `master` or `worker`.
+You can define the role for each host in the `agent-config.yaml` file by using the `role` parameter.
+If you do not assign a role to the hosts, the roles will be assigned at random during installation.
+
+.Sample agent-config.yaml file
+[source,yaml]
+----
+apiVersion: v1beta1
+kind: AgentConfig
+metadata:
+  name: example-cluster
+rendezvousIP: 192.168.111.80
+hosts:
+  - hostname: master-1
+    role: master
+    interfaces:
+      - name: eno1
+        macAddress: 00:ef:44:21:e6:a5
+  - hostname: master-2
+    role: master
+    interfaces:
+      - name: eno2
```

Comment:
nit: I would name the interface `eno1` on each host, just because it would be unusual to have 4 different servers each using a different interface.